test/e2e/app-dir/dynamic-io-errors/dynamic-io-errors.prospective-fallback.test.ts

```diff
@@ -27,6 +27,7 @@ describe(`Dynamic IO Prospective Fallback`, () => {
         // we expect the build to fail
       }
 
+      // TODO: Assert on component stack
```

Comment:
With this change:
```
Error: In Route "/blog/[slug]" this component accessed data without a fallback UI available somewhere above it using Suspense.
    at E (webpack:///src/client/components/layout-router.tsx:324:3)
    at c (webpack:///src/client/components/redirect-boundary.tsx:48:5)
    at l (webpack:///src/client/components/redirect-boundary.tsx:78:34)
    at l (webpack:///src/client/components/not-found-boundary.tsx:119:3)
    at O (webpack:///src/client/components/layout-router.tsx:466:3)
    at ErrorBoundary (webpack:///src/client/components/error-boundary.tsx:190:3)
    at P (webpack:///src/client/components/layout-router.tsx:289:3)
    at S (webpack:///src/client/components/layout-router.tsx:299:9)
    at RenderFromTemplateContext (webpack:///src/client/components/render-from-template-context.tsx:8:10)
    at R (webpack:///src/client/components/layout-router.tsx:514:9)
    at E (webpack:///src/client/components/layout-router.tsx:324:3)
    at c (webpack:///src/client/components/redirect-boundary.tsx:48:5)
    at l (webpack:///src/client/components/redirect-boundary.tsx:78:34)
    at l (webpack:///src/client/components/not-found-boundary.tsx:119:3)
    at O (webpack:///src/client/components/layout-router.tsx:466:3)
    at ErrorBoundary (webpack:///src/client/components/error-boundary.tsx:190:3)
    at P (webpack:///src/client/components/layout-router.tsx:289:3)
    at S (webpack:///src/client/components/layout-router.tsx:299:9)
    at RenderFromTemplateContext (webpack:///src/client/components/render-from-template-context.tsx:8:10)
    at R (webpack:///src/client/components/layout-router.tsx:514:9)
    at E (webpack:///src/client/components/layout-router.tsx:324:3)
    at c (webpack:///src/client/components/redirect-boundary.tsx:48:5)
    at l (webpack:///src/client/components/redirect-boundary.tsx:78:34)
    at c (webpack:///src/client/components/not-found-boundary.tsx:33:5)
    at l (webpack:///src/client/components/not-found-boundary.tsx:119:3)
    at O (webpack:///src/client/components/layout-router.tsx:466:3)
    at ErrorBoundary (webpack:///src/client/components/error-boundary.tsx:190:3)
    at P (webpack:///src/client/components/layout-router.tsx:289:3)
    at S (webpack:///src/client/components/layout-router.tsx:299:9)
    at RenderFromTemplateContext (webpack:///src/client/components/render-from-template-context.tsx:8:10)
    at R (webpack:///src/client/components/layout-router.tsx:514:9)
    at body (<anonymous>)
    at html (<anonymous>)
    at constructor (webpack://next/dist/src/client/components/redirect-boundary.tsx?07a0:48:5)
    at splitInterceptingRoute (webpack://next/dist/src/client/components/redirect-boundary.tsx?07a0:79:9)
    at splitInterceptingRoute (webpack://next/dist/src/client/components/app-router.tsx?b068:215:9)
    at constructor (webpack://next/dist/src/client/components/error-boundary.tsx?ce04:69:11)
    at errorComponent (webpack://next/dist/src/client/components/error-boundary.tsx?ce04:190:3)
    at actionQueue (webpack://next/dist/src/client/components/app-router.tsx?b068:632:3)
    at ServerInsertedHTMLProvider (webpack://next/dist/src/server/app-render/server-inserted-html.tsx?7790:14:34)
    at splitInterceptingRoute (webpack://next/dist/src/server/app-render/app-render.tsx?93c3:836:3)
```

Follow-up: 
- remove `webpack://` prefix
- Find out why names are not sourcemapped and still minified
- no useful component stack. I would've expected the component that accessed the data to show up here?